test/massive/munit/AssertTest.hx

```diff
@@ -479,6 +479,42 @@ class AssertTest
 		}
 		Assert.fail("Invalid assertion not captured");
 	}
+    
+    @Test
+    public function testThrowsStringAndObject():Void
+    {
+        // Positive case: throws expected string
+        var expectedMessage:String = "Invalid operation!";
+        var actualMessage:String = Assert.throws(function()
+        {
+            throw expectedMessage;
+        });
+        Assert.areEqual(expectedMessage, actualMessage);
+        
+        // Positive case: throws expected exception
+        var expectedError:CustomException = new CustomException('URL not reachable', 37);
+        var actualError:CustomException = Assert.throws(function()
+        {
+            throw expectedError;
+        });
+        Assert.areEqual(expectedError.message, actualError.message);
+        Assert.areEqual(expectedError.code, actualError.code);
+        
+        // Negative case: assertion if it doesn't throw
+        var failureMessage:String = "Assert.throws didn't raise an exception when nothing was thrown";
+        try {
+            Assert.throws(function()
+            {
+                var message:String = "Nothing thrown here!";
```

Comment:
It's not used. I'll rewrite it slightly (or add a comment or something) instead of keeping unused variables around.